kordac/processors/errors/Error.py

```diff
@@ -0,0 +1,5 @@
+def Error(Exception):
+    """Base class for Errors.
+    (Exceptions from external sources such as inputs).
+    """
+    pass
```

Comment:
Huh, usually GitHub would say the comment was out of date. But maybe because you can't select the EOF that it doesn't show that.